tests/sentry/api/endpoints/test_group_attachments.py

```diff
@@ -26,12 +26,17 @@ def create_attachment(self, type=None):
 
         return self.attachment
 
-    def path(self, types=None):
+    def path(self, types=None, event_ids=None):
         path = f"/api/0/issues/{self.group.id}/attachments/"
 
-        query = [("types", t) for t in types or ()]
-        if query:
-            path += "?" + urlencode(query)
+        typesQuery = [("types", t) for t in types or ()]
+        eventIdQuery = [("event_id", id) for id in event_ids or ()]
+        if typesQuery and eventIdQuery:
```

Comment:
you could just concat `typesQuery` and `eventsIdQuery` urlencode the combined list if there are any values.

Reply:
I like that idea! I'm going to look into added that in a future PR.
I'm new to backend Django, so appreciate all these tips! Thanks @AniketDas-Tekky